code/Makefile

```diff
@@ -9,7 +9,7 @@
 #############################################################################
 
 # source files in this project (main.cpp is automatically assumed)
-SOURCES := $(wildcard src/*.cpp)
+SOURCES := $(wildcard src/*.cpp) ../../../modules/i2c-library/code/src/i2c_bus.cpp
```

Comment:
this path is incorrect so it seems this should be moved to the libraries folder

Reply:
Will be fixed as soon as i2c has been merged, that way I'll be able to thoroughly test the new makefiles

Reply:
yes in theory you sould be able to use the include method we discussed:

Reply:
This has now been fixed.